libs/wallet/src/context.ts

```diff
@@ -3,7 +3,7 @@ import type {
   TransactionResponse,
 } from '@vegaprotocol/vegawallet-service-api-client';
 import { createContext } from 'react';
-import type { VegaConnector } from './connectors';
+import type { VegaConnector } from './connectors/vega-connector';
```

Comment:
chore (blocking): this is still exported from the index, so i don't think this change is needed